src/app/content/utils.ts

```diff
@@ -9,6 +9,30 @@ export { getBookPageUrlAndParams, getPageIdFromUrlParam, getUrlParamForPageId, t
 export { stripIdVersion } from './utils/idUtils';
 export { scrollSidebarSectionIntoView } from './utils/domUtils';
 
+export const cleanArchiveResponse = (archiveContent: any) => {
+  const chapterSlug = /^chapter-/;
+  const appendixSlug = /^appendix-/;
+  const chapterTitle = 'Chapter';
+  const appendixTitle = 'Appendix';
+
+  if (archiveContent.tree) {
+    archiveContent.tree.contents.map((item: any) => {
+      item.slug = item.slug.replace(chapterSlug, '').replace(appendixSlug, '');
+
+      const domNode = new DOMParser().parseFromString(item.title, 'text/html');
+      const numNode = domNode.querySelector('.os-number');
```

Comment:
This probably can be replaces with `getArchiveTreeSectionNumber` util

Reply:
its done this way so it can be modified below, the util just returns a string